temporalcli/commands.batch.go

```diff
@@ -70,6 +70,10 @@ func (c TemporalBatchListCommand) run(cctx *CommandContext, args []string) error
 	}
 	defer cl.Close()
 
+	// This is a listing command subject to json vs jsonl rules
+	cctx.Printer.StartList()
+	defer cctx.Printer.EndList()
```

Comment:
I'm thinking maybe we should use the term "array" rather than "list" in the code since that's the correct JSON terminology.

Reply:
`list` is what we call the commands and I don't want users to even have to worry about whether JSON is enabled or not. I want people to say whether their jsonl list is starting/ending too. Often people don't think of arrays as lazy/streaming. 

Having said this, if the word "array" is important enough, I can change to get PR approval.

Reply:
Thanks, that's fine, especially if there's a connection with the use of "list" as a verb in the CLI.